File: scratch/find_logic.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, '../app.js');
const content = fs.readFileSync(filePath, 'utf8');
const lines = content.split('\n');

const keywords = ['moon-icon', 'moon-phase', 'logo_luna', 'luna_', 'card-img', 'card_back', 'assets/', 'luna_creciente', 'luna_llena'];

lines.forEach((line, idx) => {
  if (keywords.some(kw => line.includes(kw))) {
    console.log(`Line ${idx + 1}: ${line.trim().substring(0, 120)}`);
  }
});

File: scratch/find_logic.py
import re

with open("../app.js", "r", encoding="utf-8") as f:
    content = f.read()

# Find occurrences of 'moon-icon' or 'moon-phase' or 'luna_'
lines = content.split('\n')
for idx, line in enumerate(lines):
    if any(keyword in line for keyword in ['moon-icon', 'moon-phase', 'logo_luna', 'luna_', 'card-img', 'card_back']):
        print(f"Line {idx+1}: {line.strip()[:120]}")
